### DISCUSSION TITLE
HA's Solarman and Solarman Smart (IOS app)

### DISCUSSION THREAD

**Discussion by liparrok** [davidrapan/ha-solarman#120] [Q&A] [answered]
Hey everyone,

no sure if this is even possible, but as soon as I added Deye in my HA Solarman integration, Solarman Smart app on my IOS phone lost connection. There are some A and B servers on */index_cn.html (* = device IP address) or server A and optional server on */config_hide.html webpage of Deye's data collector. 
So is it possible to have connected HA's and IOS app to data collector at the same time?

Many thanks,
Rok

**Comment by PV-Joe** [comment]
Hi,

This Solarman for Home Assistant runs parallel to these Apps for iOS without any problems:

![image](https://github.com/user-attachments/assets/e5e3dc29-e252-4b66-bfa8-7b1d8a3f50cd)

... for my Deye Hybrid.

**Reply by liparrok** [reply]
Hm interesting. Not sure than what happened last time, maybe because I change IP address of dongle?
Anyway, I will try that again tomorrow and let you know...

Rok

**Comment by PaulOckenden** [comment]
Running fine in parallel here. 

BTW, off topic, but Energy Ease is sooooooo much better than Solarman Smart.

**Reply by davidrapan** [reply]
And I would add that HA is way better than all of the other combined, haha. 😆

**Reply by liparrok** [reply, marked as answer]
I do agree, HA integration rules :). But yes, for some basic user, read my parents, simple app works better than HA
Obviously changing IP address of dongle, disconnect Solarman Smart app - needs to do reconfiguration :(

Many thanks for you help.
Rok
